api/migrations/20210815220241-add-bean-types.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+console.log("add bean types");
```

Comment:
Is this debug log that forgot to be removed?

Reply:
Yes :0